source/message.html

```diff
@@ -4,24 +4,72 @@
 <TITLE></TITLE>
 <SCRIPT LANGUAGE=javascript>
 <!--
-function escapeKeyPress(e) {
+// Though these should be constants, MSHTML does not support const as of 2024
+var args = window.dialogArguments;
+var copyStatusTimeout = 5000;  // milliseconds before copy status message goes away
+
+function handleKeyPress(e) {
 	e = e || window.event;
 	if(e.keyCode == 27){ // code for escape
```

Comment:
@coderabbitai could you suggest an additional condition to this if statement, which will detect if the user presses `Alt+C`, and run the `onCopyButtonPress()` function if so? It must work in the context of MSHTML (Internet Explorer 10).

Reply:
@coderabbitai that is the identical code I used in commit 6ee880cd224ca13eed671a8ff0c63549afcb93a1. It does not work to detect Alt+C in MSHTML; a Windows error sound is played if the key sequence is pressed, and it is not passed to the javascript. Can you suggest an alternative?